test/Microsoft.ML.Tests/Transformers/HashTests.cs

```diff
@@ -47,10 +47,10 @@ public void HashWorkout()
 
             var dataView = ML.Data.ReadFromEnumerable(data);
             var pipe = new HashingEstimator(Env, new[]{
```

Comment:
>HashingEstimator [](start = 27, length = 16)

why not ml.Transforms.Conversion.... ?

Reply:
Will update, since it was compiling I did not get to change that, but it's best to be consistent you are right!

---
In reply to: [253664783](https://github.com/dotnet/machinelearning/pull/2364#discussion_r253664783) [](ancestors = 253664783)